packages/react/package.json

```diff
@@ -14,7 +14,8 @@
   "files": [
     "lib/**/*",
     "es/**/*",
-    "umd/**/*"
+    "umd/**/*",
+    "telemetry.yml"
```

Comment:
@carbbon-devs we added this to the config because we think we need it for the config file to ship with the deployed package, but please confirm that 😬

Reply:
Yep I think that should do it